config/float64_test.go

```diff
@@ -6,138 +6,148 @@ import (
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(v).To(BeNumerically("~", 456.78))
+		v := AsFloat64Default(b, "<key>", 50)
+		Expect(v).To(BeNumerically("==", 50))
 	})
 
-	It("returns an error if the value cannot be parsed", func() {
+	It("panics if the value cannot be parsed", func() {
 		b := Map{"<key>": String("<invalid>")}
 
-		_, err := GetFloat64Default(b, "<key>", 456.78)
-		Expect(err).To(MatchError(`<key> is not a valid 64-bit float: strconv.ParseFloat: parsing "<invalid>": invalid syntax`))
+		Expect(func() {
+			AsFloat64Default(b, "<key>", 50)
+		}).To(PanicWith(`expected <key> to be a 64-bit floating-point number: strconv.ParseFloat: parsing "<invalid>": invalid syntax`))
 	})
 })
 
-var _ = Describe("func MustGetFloat64()", func() {
-	It("returns a positive float value", func() {
-		b := Map{"<key>": String("123.45")}
+var _ = Describe("func AsFloat64Between()", func() {
+	It("returns an float64 value", func() {
+		b := Map{"<key>": String("50")}
 
-		v, ok := MustGetFloat64(b, "<key>")
-		Expect(v).To(BeNumerically("~", 123.45))
-		Expect(ok).To(BeTrue())
+		v := AsFloat64Between(b, "<key>", -100, 100)
+		Expect(v).To(BeNumerically("==", 50))
 	})
 
-	It("returns a negative float value", func() {
-		b := Map{"<key>": String("-123.45")}
+	It("panics if the key is not defined", func() {
+		b := Map{}
 
-		v, ok := MustGetFloat64(b, "<key>")
-		Expect(v).To(BeNumerically("~", -123.45))
-		Expect(ok).To(BeTrue())
+		Expect(func() {
+			AsFloat64Between(b, "<key>", -100, 100)
+		}).To(PanicWith(`<key> is not defined`))
 	})
 
-	It("sets ok to false if the key is not defined", func() {
-		b := Map{}
+	It("panics if the value is lower than the minimum", func() {
+		b := Map{"<key>": String("-120")}
+
+		Expect(func() {
+			AsFloat64Between(b, "<key>", -100, 100)
+		}).To(PanicWith(`expected <key> to be between -100.000000 and 100.000000 (inclusive), got -120.000000`))
+	})
+
+	It("panics if the value is greater than the maximum", func() {
+		b := Map{"<key>": String("120")}
 
-		_, ok := MustGetFloat64(b, "<key>")
-		Expect(ok).To(BeFalse())
+		Expect(func() {
+			AsFloat64Between(b, "<key>", -100, 100)
+		}).To(PanicWith(`expected <key> to be between -100.000000 and 100.000000 (inclusive), got 120.000000`))
 	})
 
 	It("panics if the value cannot be parsed", func() {
 		b := Map{"<key>": String("<invalid>")}
 
 		Expect(func() {
-			MustGetFloat64(b, "<key>")
-		}).To(PanicWith(
-			MatchError(`<key> is not a valid 64-bit float: strconv.ParseFloat: parsing "<invalid>": invalid syntax`),
-		))
+			AsFloat64Between(b, "<key>", -100, 100)
+		}).To(PanicWith(`expected <key> to be a 64-bit floating-point number: strconv.ParseFloat: parsing "<invalid>": invalid syntax`))
 	})
 })
 
-var _ = Describe("func MustGetFloat64Default()", func() {
-	It("returns a positive float value", func() {
-		b := Map{"<key>": String("123.45")}
+var _ = Describe("func AsFloat64DefaultBetween()", func() {
+	It("returns an float64 value", func() {
```

Comment:
Is this correct grammar? Should it be "returns a float64 value"?
> returns an float64 value

Reply:
Fixed.